### ISSUE TITLE
Use U-ADS tools to build OGC application package for the extended BCDP notebook.

### ISSUE THREAD

**Issue by jjacob7734** [unity-sds/unity-analytics-bcdp#22]
Consult with the U-ADS team to understand the Unity capabilities to build OGC applications.  A key repo for this is https://github.com/unity-sds/unity-app-generator.  Use the Unity tools to build an OGC application package out of the extended BCDP notebook, including workflow CWL, 3-step stage-in, process, and stage-out CWL, and associated Docker images.

**Comment by mike-gangl** [comment]
also add:

* Register in the dockstore catalog

And either add, or create a second ticket for:

* Deploying the applicaiton package/ workflow to an SPS

**Comment by mike-gangl** [comment]
@jjacob7734 , @agoodm  Can we generate the ApplicationPackage with @mcduffie (ADS) based code?

**Comment by jjacob7734** [comment]
> @jjacob7734 , @agoodm Can we generate the ApplicationPackage with @mcduffie (ADS) based code?

@mike-gangl Yes, @agoodm will try https://github.com/unity-sds/unity-app-generator

**Comment by jjacob7734** [comment]
@agoodm reported he was able to use the U-ADS tools to build the CWL and application descriptor JSON, but they don't yet support the container building and hosting on a registry.  For now he is using Max Zhan's CI/CD implementation for MAAP-HEC project to build the container and upload it to Docker Hub.
